content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -130,119 +103,97 @@ spec:
-        --set components.cni.namespace=kube-system
-    {{< /text >}}
-
-## Sidecar injection compatibility
-
-The use of the Istio CNI plugin requires Kubernetes pods to be deployed with a sidecar injection method
-that uses the `istio-sidecar-injector` configmap created from the installation with the
-`--set cni.enabled=true` option.  Refer to [Istio sidecar injection](/docs/setup/additional-setup/sidecar-injection/)
-for details about Istio sidecar injection methods.
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  components:
+    cni:
+      enabled: true
+      namespace: kube-system
+  values:
+    sidecarInjectorWebhook:
+      injectedAnnotations:
+        k8s.v1.cni.cncf.io/networks: istio-cni
+    cni:
+      cniBinDir: /var/lib/cni/bin
+      cniConfDir: /etc/cni/multus/net.d
+      cniConfFileName: istio-cni.conf
+      chained: false
+{{< /text >}}
 
-The following sidecar injection methods are supported for use with the Istio CNI plugin:
+## Operation details
 
-1.  [Automatic sidecar injection](/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
-1.  Manual sidecar injection with the `istio-sidecar-injector` configmap
-    1.  [`istioctl kube-inject`](/docs/reference/commands/istioctl/#istioctl-kube-inject) using the configmap directly:
+### Upgrade
 
-        {{< text bash >}}
-        $ istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml --injectConfigMapName istio-sidecar-injector
-        $ kubectl apply -f deployment-injected.yaml
-        {{< /text >}}
+When upgrading Istio with [in-place upgrade](/docs/setup/upgrade/in-place/),
+CNI component can be upgraded together with the control plane using one `IstioOperator` resource.
 
-    1.  `istioctl kube-inject` using a file created from the configmap:
+When upgrading Istio with [canary upgrade](/docs/setup/upgrade/canary/), since CNI component runs as a cluster singleton,
+it is recommended to operate and upgrade CNI separately from the revisioned control plane.
+The following `IstioOperator` can be used to operate CNI component independently.
 
-        {{< text bash >}}
-        $ kubectl -n istio-system get configmap istio-sidecar-injector -o=jsonpath='{.data.config}' > inject-config.yaml
-        $ istioctl kube-inject -f deployment.yaml -o deployment-injected.yaml --injectConfigFile inject-config.yaml
-        $ kubectl apply -f deployment-injected.yaml
-        {{< /text >}}
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  profile: empty # Do not include other components
+  components:
+    cni:
+      enabled: true
+  values:
+    cni:
+      excludeNamespaces:
+        - istio-system
+        - kube-system
+{{< /text >}}
 
-## Operational details
+When installing revisioned control plane with CNI component enabled,
+`values.istio_cni.enabled` needs to be set, so that sidecar injector does not `istio-init` `initContainer`.
```

Comment:
```suggestion
When installing revisioned control planes with the CNI component enabled,
`values.istio_cni.enabled` needs to be set, so that sidecar injector does not inject the `istio-init``initContainer`.
```